tests/suites/0_stateless/03_dml/03_0004_select_order_by.sql

```diff
@@ -21,10 +22,11 @@ INSERT INTO t2 VALUES(1, parse_json('{"k":"v"}')),
                      (3, parse_json('[1,2,3]')),
                      (4, parse_json('10')),
                      (5, parse_json('null')),
-                     (6, parse_json('true'));
+                     (6, parse_json('true')),
+                     (7, NULL );
 
 SELECT id, var FROM t2 ORDER BY var ASC;
-SELECT id, var FROM t2 ORDER BY var DESC;
+SELECT id, var FROM t2 ORDER BY var DESC NULLS FIRST;
```

Comment:
You should add the test in `databend/tests/suites/0_stateless/20+_others/20_0001_planner_v2.sql`.

Reply:
@xudong963 same error, I already set new planner env